src/hamster-windows-service.py

```diff
@@ -41,6 +41,14 @@ def _open_window(self, name):
         subprocess.run("hamster {} &".format(name),
                        shell=True)
 
+    @dbus.service.method("org.gnome.Hamster.WindowServer")
+    def edit(self, id=None):
+        if id:
+            # Need an edit <id> command in cli
+            raise NotImplementedError()
```

Comment:
Sorry this won't do it. The Shell extension also uses edit with <id> (that's the all-important button I'm using all the time to fix my wrong start times ...)

If you want to maintain compatibility, you need to revert 0cffe9c2c7e669a3b0557a15f61e2a8ea538be21.

Reply:
That's not helping, on the contrary.
Please read carefully the links before posting: https://github.com/projecthamster/hamster/issues/548#issuecomment-584676139